crates/sui-core/src/consensus_handler.rs

```diff
@@ -131,14 +131,16 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
         /* (serialized, transaction, output_cert) */
         let mut transactions = vec![];
         let timestamp = consensus_output.sub_dag.commit_timestamp;
+        let leader_author = consensus_output.sub_dag.leader.header().author();
 
         let epoch_start = self
             .epoch_store
             .epoch_start_config()
             .epoch_start_timestamp_ms();
         let timestamp = if timestamp < epoch_start {
             error!(
-                "Unexpected commit timestamp {timestamp} less then epoch start time {epoch_start}"
+                "Unexpected commit timestamp {timestamp} less then epoch start time {epoch_start}, author {leader_author}, round {round}",
+
```

Comment:
Let's keep in mind that the author here now will be the id we use in Narwhal - meaning an int between `0..num_of_validators`